src/opening-period/time-span/Weekdays.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import './Weekdays.scss';
+
+export default function Weekdays({
+  index,
+  register,
+}: {
+  index: number;
+  register: Function;
+}): JSX.Element {
+  return (
+    <fieldset>
+      <legend>Päivät *</legend>
+      <label
+        data-test={`weekdays-monday-${index}`}
+        htmlFor={`timeSpans[${index}].weekdays[0]`}
+        className="day-label">
+        <input
+          ref={register()}
+          type="checkbox"
+          name={`timeSpans[${index}].weekdays[0]`}
+          className="opening-hours-day-checkbox-input"
+          id={`timeSpans[${index}].weekdays[0]`}
+        />
+        <span className="day-option">Ma</span>
```

Comment:
It might be easier to maintain this if there would be a DayCheckBox-component, etc.
Then it would be just:
```js
<fieldset>
      <legend>Päivät *</legend>
      <DayCheckbox day="monday"/>
      <DayCheckbox day="tuesday"/>
      ....
</fieldset>
```

Reply:
It did not focus on these day-checkboxes when I tested navigating the form with just tab 🤔

Reply:
Clearly a bug, I'll try to fix that, good find!

- [ ] Enable focus to the day-checkboxes when navigating by tab

Reply:
> It might be easier to maintain this if there would be a DayCheckBox-component, etc.
> Then it would be just:
> 
> ```js
> <fieldset>
>       <legend>Päivät *</legend>
>       <DayCheckbox day="monday"/>
>       <DayCheckbox day="tuesday"/>
>       ....
> </fieldset>
> ```

Good idea, I'll extract that as a separate component!

- [ ] Extract DayCheckBox as a separate component